src/frontend/handlers.go

```diff
@@ -72,6 +72,21 @@ func (fe *frontendServer) homeHandler(w http.ResponseWriter, r *http.Request) {
 		ps[i] = productView{p, price}
 	}
 
+	if os.Getenv("SRE_RECIPE_1") == "Active" {
```

Comment:
This approach tightly couples the recipes with the code of the app. It also makes discovering the problem easier for the users using the Sandbox.
Alternatively, you could pretend that our frontend has a few feature flags. Each flag corresponds to a feature we want to roll out. I.g. currency conversion on the frontend being one such feature.
Then your logic would still apply. You can try looking at this approach here: https://cloudnativeclojure.org/feature-flags/annotations/2019/03/03/feature-flags.html

Reply:
I've renamed the flag but I'm not sure if I've addressed this comment in its entirety. Are you suggesting that we use the Kubernetes downwards API? or did you want the name of the flag to be less obvious to users who would look at the code?

Reply:
The minimum - the flag should be less obvious. We can also make new sre-recipes in the future that use multiple feature flags turned on/off.

The maximum - perhaps we can use downwards API so we can use labels for scoping a feature target. (You can do it step by step - do the minimum, get it to work and then consider doing maximum in the new PR if you have time)

Reply:
Okay I see - I think the flag works and is not too obvious now. I'll work on utilizing the downwards API for the next PR.